src/components/validation/resources/http.rs

```diff
@@ -284,6 +284,12 @@ fn spawn_output_http_server(
     // First, we'll build and spawn our HTTP server.
     let decoder = codec.into_decoder()?;
 
+    // Note that we currently don't differentiate which events should and shouldn't be rejected-
+    // we reject all events in this server if any are marked for rejection.
+    // In the future it might be useful to be able to select which to reject. That will involve
+    // adding logic to the test case which is passed down here, and to the event itself. Since
+    // we can't guarantee the order of events, we'd need a way to flag which ones need to be
+    // rejected.
```

Comment:
Toby had mentioned this in a prior PR and I forgot tot add a comment about it in that branch, so adding it here.